gma/integration_test/src/integration_test.cc

```diff
@@ -1238,166 +1288,182 @@ TEST_F(FirebaseGmaTest, TestAdView) {
-      "SetPosition(TopLeft)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionTopLeft);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  WaitForCompletion(
-      ad_view->SetPosition(firebase::gma::AdView::kPositionTopRight),
-      "SetPosition(TopRight)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionTopRight);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  WaitForCompletion(
-      ad_view->SetPosition(firebase::gma::AdView::kPositionBottom),
-      "SetPosition(Bottom)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionBottom);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  WaitForCompletion(
-      ad_view->SetPosition(firebase::gma::AdView::kPositionBottomLeft),
-      "SetPosition(BottomLeft)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionBottomLeft);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  WaitForCompletion(
-      ad_view->SetPosition(firebase::gma::AdView::kPositionBottomRight),
-      "SetPosition(BottomRight)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionBottomRight);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  // Move to some coordinates.
-  WaitForCompletion(ad_view->SetPosition(100, 300), "SetPosition(x0, y0)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionUndefined);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  WaitForCompletion(ad_view->SetPosition(100, 400), "SetPosition(x1, y1)");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(ad_view->bounding_box().position,
-            firebase::gma::AdView::kPositionUndefined);
-  EXPECT_EQ(++expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
-
-  // Try hiding and showing the AdView.
-  WaitForCompletion(ad_view->Hide(), "Hide 1");
-  PauseForVisualInspectionAndCallbacks();
-  EXPECT_EQ(expected_num_bounding_box_changes,
-            bounding_box_listener.bounding_box_changes_.size());
+  // Suppress the extensive testing below if the ad failed to load.
+  if (ad_loaded) {
+    EXPECT_EQ(ad_view->ad_size().width(), kBannerWidth);
```

Comment:
This section didn't diff properly. The internals of this if block are the same as the original, but they're now indented and reformatted accordingly.